graphql/schemas/Ecosystem/modules.graphql

```diff
@@ -0,0 +1,9 @@
+type Module {
```

Comment:
@FredPeal change this to KanvasModule so this doesn't affect other places in the future